common/changes/@microsoft/rush/dotenv_2025-03-10-22-13.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Does `rushx` support this feature as well?

Reply:
It does now.